lib/web/integrations_awsoidc.go

```diff
@@ -372,6 +377,132 @@ func (h *Handler) awsOIDCListSecurityGroups(w http.ResponseWriter, r *http.Reque
 	}, nil
 }
 
+// awsOIDCListSecurityGroups returns a map of required VPC's and its subnets.
```

Comment:
I think we should add more information about what this endpoint does and where it is used.